src/elm/Claim.elm

```diff
@@ -475,6 +500,172 @@ viewClaimCard loggedIn profileSummaries claim =
         ]
 
 
+type alias CompletionStatus =
+    { approved : Int
+    , disapproved : Int
+    , verifications : Int
+    }
+
+
+type VotingProgressBar
+    = ClaimCard
+    | ClaimModalAndDetailPage
+
+
+viewVotingProgress : Shared -> CompletionStatus -> VotingProgressBar -> Html msg
+viewVotingProgress shared completionStatus claimSpace =
+    let
+        { t } =
+            shared.translators
+
+        totalVotes =
+            toFloat completionStatus.verifications
+
+        approvedWidth =
+            (toFloat completionStatus.approved / totalVotes) * 100
+
+        disapprovedWidth =
+            (toFloat completionStatus.disapproved / totalVotes) * 100
+
+        votingLeft =
+            completionStatus.verifications - completionStatus.approved - completionStatus.disapproved
+
+        voteNumberTitleConditional compStatus =
+            if compStatus == 1 then
+                text (t "claim.vote")
+
+            else
+                text (t "claim.votes")
+
+        viewVotesBar =
+            div [ class "w-full h-2 bg-gray-500 flex rounded-full my-2" ]
+                [ div
+                    [ class "flex rounded-full overflow-hidden h-2"
+                    , style "width" (String.fromFloat (approvedWidth + disapprovedWidth) ++ "%")
+                    ]
+                    [ div [ class "bg-green", style "width" (String.fromFloat (toFloat completionStatus.approved / toFloat (completionStatus.approved + completionStatus.disapproved) * 100) ++ "%") ] []
+                    , div
+                        [ class "bg-red"
+                        , style "width" (String.fromFloat (toFloat completionStatus.disapproved / toFloat (completionStatus.approved + completionStatus.disapproved) * 100) ++ "%")
+                        ]
+                        []
+                    ]
+                ]
+    in
+    case claimSpace of
+        ClaimCard ->
```

Comment:
I feel like we could use this `case` inside the HTML itself (once to check if we should show the numbers under the bars, and another time to check if we should show the approved/disapproved votes on the modal or details page), since both branches are pretty similar, with only a couple of differences. (You don't need to change this)